### PR TITLE
chore(ci): remove extra push step

### PR DESCRIPTION
**Pull request by castrojo**
The merge queue handles this

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -3,8 +3,6 @@ on:
   schedule:
     - cron: '20 22 * * *'  # 10:20pm everyday
   push:
-    branches:
-      - main
```
Comment (p5) [inline .github/workflows/build.yml:7]: Wouldn't this run the workflow on all pushes to all branches?  Inc PR branches?

Comment (castrojo) [reply .github/workflows/build.yml:7]: It's hard to explain, the way it's working in main is the `workflow_dispatch` is handling both the PRs and merge to main separately and in order.

Comment (castrojo) [reply .github/workflows/build.yml:7]: If you want to queue up some changes and pile up a handful of PRs to watch it in realtime I've found that to be useful!

Comment (p5) [reply .github/workflows/build.yml:7]: Well, this PR pushed a new `:37` image :smile: 

https://github.com/ublue-os/bluefin/actions/runs/4821382578/jobs/8587179489#step:9:71

Comment (p5) [reply .github/workflows/build.yml:7]: I'm checking in `main` as I may be misunderstanding something

**Review by p5 — APPROVED** [review]
LGTM

**Pull request merged**